src/components/dropdown/Dropdown.tsx

```diff
@@ -1,10 +1,12 @@
+import * as classnames from 'classnames';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 
 export interface IDropdownOwnProps extends React.ClassAttributes<Dropdown> {

```

Comment:
I get an error saying the class cannot extend both because the definition of the ref property is different in the two: `Named property 'ref' of types 'ClassAttributes<Dropdown>' and 'HTMLProps<HTMLDivElement>' are not identical.`